src/generatePOT/test/__snapshots__/generatePOT.spec.js.snap

```diff
@@ -1,10 +1,14 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`generatePOT should return right POT content 1`] = `
-"msgid \\"All work is done!\\"
+"#: index.js:13
+msgid \\"All work is done!\\"
 msgstr \\"\\"
+#: ../src/controller/runner.js:31
+#: ../src/controller/runner.js:45
```

Comment:
The reason is to have the context of the message. I think its not a big increase of the file size, but if you have to fix something, it can be really helpful. For example, if 2 messages are the same like: `Something`, `Something.`

Reply:
some tools let's you link a code base to the translations

Reply:
so at some points we may need to make the file path relative to the project root instead of relative to the tool path

Reply:
It is relative to the `pot` file. But it can be changed, but in this case somebody should pass the project root. Maybe it can be a parameter of the rollup plugin.

Reply:
let's keep this as a possible future improvement and try with  real tools first.